src/tr2/global/vars_decomp.h

```diff
@@ -12,7 +12,7 @@
 #define g_RhwFactor (*(float*)0x0046408C) // = 335544320.0f
 #define g_CineTrackID (*(int32_t*)0x004640B0) // = 1
 #define g_CineTickRate (*(int32_t*)0x004640B8) // = 0x8000
-#define g_CD_TrackID (*(int16_t*)0x004640BC) // = -1
+#define g_Legacy_CD_TrackID (*(int16_t*)0x004640BC) // = -1
```

Comment:
Hi, this file is updated by progress.txt as well. Let's mark this variable with a `+` flag (without renaming it). This way, it'll be visible as `g_CD_TrackID` in IDA, while it'll be completely skipped in our codebase.

As for `Music_Legacy_Play`, we can safely remove its definitions altogether, considering how since 0.2 we're no longer trying to inject it (and thus are free to use our own signatures).

Reply:
Thank you, and no - my apologies for not getting this bit right.

I've changed the variable back to OG and marked it as decompiled.

The legacy function is still in use though: below is with logging added, it's triggered when Lara completes the assault course (possibly elsewhere, but this one for sure). I'm assuming that flip effect does some comparison with the player's time and decides which track to play (22 is "gosh, that was my best time yet").

```
game/music/music_main.c 116 Music_Legacy_Play Legacy call for track: 22
game/music/music_main.c 149 Music_Play Playing track 22 (real: 19), mode: 0
```

![image](https://github.com/user-attachments/assets/1e9268c1-57d9-439f-92c2-b6677798b54a)